pycolmap/README.md

```diff
@@ -23,12 +23,10 @@ not built with CUDA support, which requires building from source.
 3. Build PyCOLMAP:
   - On Linux and macOS:
 ```bash
-cd pycolmap
 python -m pip install .
 ```
   - On Windows, after installing COLMAP [via VCPKG](https://colmap.github.io/install.html#id3), run in powershell:
 ```powershell
-cd pycolmap
 python -m pip install . `
```

Comment:
Are these changes really correct here? Don't we need to either cd into the pycolmap folder or otherwise run `python -m pip install pycolmap`?